src/sqlfluff/rules/L006.py

```diff
@@ -145,15 +146,27 @@ def _eval(self, context: RuleContext) -> EvalResultType:
                         )
                         check_before = True
                     if self.matches_target_tuples(trailing, self._target_elems):
-                        after_anchor = trailing
-                        self.logger.debug(
+                        after_anchor = trailing  # pragma: no cover
+                        self.logger.debug(  # pragma: no cover
```

Comment:
Are there examples of tests cases that would provide coverage here? TBH, not sure why this is no longer covered with changes made

Reply:
I don't know which test cases, but you could try [this feature](https://coverage.readthedocs.io/en/6.3.2/contexts.html#dynamic-contexts) of the coverage package we use for measuring coverage. It should be able to tell you which tests use which code.

I haven't used the technique above, but you can also do something similar, manually:
Switch back to the main branch and set a breakpoint on these lines, or log a stack trace. That'll tell you which test cases are executing these lines.